fe/fe-core/src/main/java/com/starrocks/storagevolume/StorageVolume.java

```diff
@@ -65,6 +68,7 @@ public StorageVolume(String name, String svt, List<String> locations,
             throw new AnalysisException("Storage params is not valid");
         }
         this.params = params;
+        this.id = GlobalStateMgr.getCurrentState().getNextId();
```

Comment:
It is not a good way to allocate id in Constructor, because this class will depend on the IdGenerator, but that is not necessary.
You need to pass the id in the arguments.

Reply:
> It is not a good way to allocate id in Constructor, because this class will depend on the IdGenerator, but that is not necessary. You need to pass the id in the arguments.

I have moved it to `StorageVolumeMgr`.